packages/content/eleventy.config.dev.cjs

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+const pathlib = require('path');
+
+module.exports = (eleventyConfig) => {
+  symlinkForce('../assets', '_dev/assets');
+  symlinkForce('../../client/lib/entrypoints', '_dev/js');
+  return {
+    dir: {
+      input: 'site',
+      output: '_dev',
+    },
+  };
+};
+
+function symlinkForce(target, path) {
```

Comment:
Add comment on why this is here?

Reply:
Done

Reply:
Did you upload the commit?

Reply:
I see the comment in #1335... that's fine.